python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,13 +41,12 @@ def _getGangaList():
 
 
 class Node(object):
-    _ref_list = ['_parent', '_registry', '_index_cache', '_proxyObject']
+    _ref_list = ['_parent', '_registry', '_index_cache']
```

Comment:
This is used to prevent any copy operation involving ```__dict__``` from going in infinite loops. Is this safe?